apps/src/weblab2/CDOIDE/center-pane/InternalEditor.tsx

```diff
@@ -0,0 +1,99 @@
+// This whole thing is shut off, though still present, because I can't get @uiw/react-codemirror to
```

Comment:
Should we remove this and file a follow-up ticket to investigate adding a React CodeMirror alternative to the main repo? We could leave this in a separate branch or some other idea?

Reply:
I noted above that I'll yank out all the code we're not currently using once this PR is in place and I can compartmentalize it into a nice deleting PR. So for sure can delete it, just as a follow up.